repos/spack_repo/builtin/packages/py_tokenizers/package.py

```diff
@@ -24,7 +25,8 @@ class PyTokenizers(PythonPackage):
 
     # TODO: This package currently requires internet access to install.
     depends_on("py-maturin@1", when="@0.14:", type="build")
```

Comment:
FWIW. Based on prior comments from Adam, I'm ignoring the upper bound for `py-maturin` in the `pyproject.toml` file.